lib/import_users.rb

```diff
@@ -56,7 +58,13 @@ def read
   end
 
   def create_user(username, password_digest, title, first_name, last_name, full_name, email_address)
-    person = Person.create!
+    # Check whether the user is already in the database
+    ea = EmailAddress.find_by_value(email_address)
```

Comment:
if we don't use verified emails, there's an outside chance that we'd link different people together.